source/common/http/conn_manager_impl.h

```diff
@@ -303,6 +303,7 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     void decodeHeaders(HeaderMapPtr&& headers, bool end_stream) override;
     void decodeData(Buffer::Instance& data, bool end_stream) override;
     void decodeTrailers(HeaderMapPtr&& trailers) override;
+    void decodeMetadata(MetadataMap&) override { NOT_REACHED_GCOVR_EXCL_LINE; }
```

Comment:
Do we think there should be anything done if an HTTP/1.1 codec is asked to encode metadata?  If nothing else maybe error reporting stats so folks know their metadata is being swallowed?

Reply:
Good call! I added an error log in Http1::StreamEncoderImpl::encodeHeader().